src/docker/runner/image_group.go

```diff
@@ -0,0 +1,8 @@
+package runner
+
+// ImageGroup is a group of images to run at once
+type ImageGroup struct {
+	ID          string
+	Names       []string
```

Comment:
I think we have to give this field a better name. This is really the key for a service/dependency in `docker-compose.yml` right? `ImageGroup.Names` makes me think of docker image names which are like `tmp_user-service`. Docker-compose calls those keys service names, that wouldn't be a bad name although I'm not sure if that will cause confusion with what we call service names.

Reply:
How about `containerNames` since they ended up being used as the name of the container?

Reply:
I think technically they're still different from containerNames for the regular services right? Those service container names are prefixed. That would also be confusing because we don't list a container name for services in `docker-compose.yml`, what we're really referring to are the keys

Reply:
The image names are prefixed with the compose project name, but the container names match the keys in the `docker-compose.yml`. I figured this out when creating the get exit code function.